packages/utils/src/namespace.ts

```diff
@@ -276,3 +278,19 @@ export function isNamespace(value: unknown): value is Namespace {
 export function isNamespacesObject(value: unknown): value is Namespaces {
   return is(value, NamespacesStruct);
```

Comment:
why not just merge the two functions?

```typescript
import { assert } from 'superstruct';

export function assertNamespacesObject(value: unknown): asserts value is Namespaces {
  return assert(value, NamespacesStruct);
}
```

Reply:
In this case we want to throw a JSON-RPC error, rather than a `StructError`. We could maybe have a more generic way to do that?

Reply:
rethrow, or up to you?